docs/models.rst

```diff
@@ -86,6 +86,54 @@ The default parameters are optimized for SO-SAT observations. Meaning that the a
+
+The Stokes I map is computed from the template one as it follows: 
+
+if target Nside <= 512:
+    
+    #. The template map at a ``nside=512``  is downgraded at the target nside 
+
+if target Nside > 512 :
+    
+    #. The template map at a ``nside=2048``  is downgraded(eventually upgraded) at the target nside
+
+Q and U maps can be computed from the template CO emission  map, :math:`I_{CO}`,  assuming a constant  fractional polarization, as: 
+
+.. math::
+
+    Q = f_{pol} I_{CO}  g_d \cos( 2 \psi) 
+
+    U  = f_{pol} I_{CO}  g_d \sin( 2 \psi) 
+
+with :math:`g_d` and :math:`\psi` being respectively the depolarization and polarization angle maps estimated from a dust map as : 
+
+.. math::
+
+    g_d = \frac{ \sqrt{Q^2_{d,353}    + U^2_{d,353}   } }{f_{pol} I_{d,353} } 
+ 
+    \psi = \frac{1}{2} \arctan {\frac{U_{d,353}}{Q_{d,353}}}
+
+
+Most of the CO emission is expected to be confined in the  Galactic midplane. However, there are still regions at high Galactic latitudes  where the CO emission has been purely assessed (by current surveys) and where the Planck signal-to-noise was not enough to detect any emission.
+
+The PySM user can include the eventuality of molecular emission (both unpolarized and polarized) at High Gal. Latitudes by coadding to the emission maps one realization of CO emission simulated with MCMole3D together with  the Planck CO map. The polarization is simulated similarly as above. 
+
+The ``MCMole3D`` input parameters  are are obtained from best fit with the Planck CO 1-0 map (see Puglisi et al. 2017 and the `documentation <http://giuspugl.github.io/mcmole/index.html>`). If ``include_high_galactic_latitude_clouds=True``, a mock CO cloud map is simulated with ``MCMole3D`` and coadded to the Planck CO emission map. The polarization is simulated similarly as above.
+
+The installation of ``mcmole3d`` is not required, HGL clouds can be input to the CO emission by setting ``run_mcmole3d=False``  (which is the default). However, if one wants to run several mock CO  realizations observing high Galactic latitude patches we encourage to run ``mcmole3d`` by changing ``random_seed`` in the CO class constructor. The parameter ``theta_high_galactic_latitude_deg`` set the latitude above which CO emission from high Galactic latitudes can be included. 
+
+The default parameters are set to include  CO 1-0 emission and polarization (with 0.1% constant polarization fraction), in particular: 
+
+    #. ``polarization_fraction= 0.001``, on average is the expected level on 10% regions of the sky. However, polarization from CO emission have been detected at larger fluxes in  Orion and Taurus complexes (Greaves et al.1999 ) 
+    #. ``theta_high_galactic_latitude_deg = 20``, includes CO emission at :math:`|b|>\theta_{hgl}` from one realization of mcmole3d maps. Be aware that the larger :math:`theta_{hgl}`,  the farther is the  Galactic plane and  the more unlikely is to find high Galactic latitude clouds.  
```

Comment:
does `theta_high_galactic_latitude_deg` have any impact if you do not run `mcmole3d`? if not, what value have you used for the run which is included in the package?

Reply:
Right, ``theta_hgl``  doesn't impact the sim ! For that run i've used the default one,  ``theta_hgl = 20 deg``.

Reply:
ok, please add this as well to the docs